ReactNativeTemplate/android/app/build.gradle

```diff
@@ -109,6 +109,11 @@ android {
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
+        debug {
+            defaultConfig {
+                targetSdkVersion 22
```

Comment:
@ivanbogdanov We want this to be `25` as well. Setting this to `22` is not solving the problem, it's simply supporting a version before the new security popup feature was introduced in `Android`. We need the `targetSdkVersion` to be the same for both `debug` and `release`. I would revert the changes to this file. We need to fix this while keeping `targetSdkVersion` as `25`.